ansible/playbooks/roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -35,12 +35,12 @@ node:
     kafka_api:
       brokers:
 {% for host in groups["redpanda"] %}
-      - {{ hostvars[host]['inventory_hostname' if (hostvars[host].advertise_public_ips | d() | bool) else 'private_ip'] }}:{{ redpanda_kafka_port }}
+      - {{ hostvars[host]['private_ip'] }}:{{ redpanda_kafka_port }}
```

Comment:
cannot advertise public ip here

Reply:
-1
Why wouldn't this be possible? In AWS this is how you need to do things for public IPs.

Reply:
After testing in AWS, using a public IP here causes rpk clients on the broker to use the public IP rather than the private one when communicating to the Redpanda process on the same broker while advertising public ip. While this can work, it seems odd to route traffic on a node outside the network and then back in.

This section is only used for rpk on the broker (per Travis B.) so setting public ip was not useful for other purposes.

Reply:
Yeah, I don't think it's a big deal either way. What we ought to do is move the client section out into a client role. It's useful for this to work when you're logged into a broker, but it's more useful to have an actual client config that we can deploy to other servers.

Reply:
Not a bad idea. I'll make a note to add that in a later round of work.

I've added some convenience scripting (that could do with a cleanup) to aid in rapid testing and standup that includes some one liners to generate REDPANDA_BROKERS and the like, but it'd make more sense to do something more complete.